ocs_ci/deployment/helpers/hypershift_base.py

```diff
@@ -62,6 +64,49 @@ def wrapper(self, *args, **kwargs):
     return wrapper
 
 
+@switch_to_orig_index_at_last
+def get_cluster_vm_namespace(cluster_name=None):
+    """
+    Get the cluster virtual machines namespace by the cluster name
+
+    Args:
+        cluster_name (str): The cluster name.
+
+    Returns:
+        str: The cluster virtual machines namespace
+
+    """
+    cluster_name = cluster_name or config.ENV_DATA["cluster_name"]
+    pattern = f"clusters-{cluster_name}"
+    config.switch_to_provider()
+    cluster_vm_namespaces = get_namespce_name_by_pattern(pattern=pattern)
+    assert (
+        cluster_vm_namespaces
+    ), f"Didn't find the cluster namespace for the cluster {cluster_name}"
+
+    return cluster_vm_namespaces[0]
+
+
+@switch_to_orig_index_at_last
+def get_hosted_cluster_type(cluster_name=None):
+    """
+    Get the hosted cluster type
+
+    Args:
+        cluster_name (str): The cluster name
+
+    Returns:
+        str: The hosted cluster type in lowercase
+
+    """
+    cluster_name = cluster_name or config.ENV_DATA["cluster_name"]
+    config.switch_to_provider()
+    ocp_hosted_cluster_obj = OCP(
+        kind=constants.HOSTED_CLUSTERS, namespace="clusters", resource_name=cluster_name
```

Comment:
Do you think it would make sense to add "clusters" to the constants file?

Reply:
Yes, I was wondering if I should add it to the constants file.